src/test/resources/no/nav/familie/ef/sak/beregning_barnetilsyn_en_mnd.feature

```diff
@@ -0,0 +1,81 @@
+      | 7   | 500.13        | 300.13              | 0                  | 3          | 01.2022     |                 |
+      | 8   | 7000          | 0                   | 70                 | 3          | 01.2022     |                 |
+      | 9   | 7000          | 0                   | 70                 | 2          | 01.2022     |                 |
+      | 10  | 7000          | 0                   | 70                 | 1          | 01.2022     |                 |
+      | 11  | 3200          | 0                   | 0                  | 1          | 01.2022     |                 |
+      | 12  | 5000          | 0                   | 0                  | 1          | 01.2022     |                 |
+      | 13  | 7000          | 0                   | 0                  | 1          | 01.2022     |                 |
+      | 14  | 6000          | 6000                | 0                  | 1          | 01.2022     |                 |
+      | 15  | 6000          | 1500                | 0                  | 1          | 01.2022     |                 |
+      | 16  | 6000          | 1500                | 980                | 1          | 01.2022     | ??              |
+      | 17  | 6400          | 1500                | 0                  | 2          | 01.2022     |                 |
+      | 18  | 9000          | 0                   | 0                  | 2          | 01.2022     |                 |
+      | 19  | 9600          | 0                   | 0                  | 3          | 01.2022     |                 |
+      | 20  | 12800         | 1500                | 0                  | 4          | 01.2022     |                 |
+      | 21  | 12800         | 0                   | 0                  | 4          | 01.2022     |                 |
+      | 22  | 12800         | 0                   | 0                  | 4          | 12.2021     |                 |
+      | 23  | 9000          | 0                   | 0                  | 3          | 12.2021     |                 |
+      | 24  | 12000         | 7500                | 0                  | 3          | 01.2022     |                 |
+      | 25  | 3600          | 6000                | 0                  | 1          | 01.2022     |                 |
+      | 26  | 7000          | 4500                | 0                  | 1          | 01.2022     |                 |
+      | 27  | 3500          | 0                   | 2940               | 1          | 01.2022     | ??              |
+      | 28  | 8000          | 0                   | 0                  | 2          | 01.2022     |                 |
+      | 29  | 11200         | 0                   | 0                  | 4          | 01.2022     |                 |
+      | 30  | 16000         | 0                   | 0                  | 4          | 01.2022     |                 |
+
+
+    Når vi beregner barnetilsyn beløp
+
+    Så forventer vi barnetilsyn periodebeløp
+      | Rad | Beløp  |
+      | 1   | 6284   |
```

Comment:
Nå som det ble så mange eksempler, ville det nok vært veldig fint med scenariomal, slik at man kan få både testvariabler og expected value på samme rad

Reply:
Det er sant! Vi hadde veldig lyst til å gjøre dette - hadde vært gull når vi snakket med funksjonelle

Reply:
...men jeg vet jo ikke helt hva en scenariomal er. ;)

Reply:
Kan vi merge cucumber-testene slik de er nå og sette opp en "forbedre cucumbertesterøkt i løpet av uka". Låner gjerne deg litt @olekvernberg for å se litt på hva en scenariomal-test er?

Reply:
Ja, selvfølgelig ikke noe stress det :)